repositories.bzl

```diff
@@ -0,0 +1,559 @@
+        )
+
+        native.bind(
+            name = "cc_wkt_protos",
+            actual = "@protobuf_git//:cc_wkt_protos",
+        )
+
+        native.bind(
+            name = "cc_wkt_protos_genproto",
+            actual = "@protobuf_git//:cc_wkt_protos_genproto",
+        )
+
+        native.bind(
+            name = "protobuf_compiler",
+            actual = "@protobuf_git//:protoc_lib",
+        )
+
+def googletest_repositories(bind=True):
+    BUILD = """
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+#
+
+cc_library(
+    name = "googletest",
+    srcs = [
+        "googletest/src/gtest-all.cc",
+        "googlemock/src/gmock-all.cc",
+    ],
+    hdrs = glob([
+        "googletest/include/**/*.h",
+        "googlemock/include/**/*.h",
+        "googletest/src/*.cc",
+        "googletest/src/*.h",
+        "googlemock/src/*.cc",
+    ]),
+    includes = [
+        "googlemock",
+        "googletest",
+        "googletest/include",
+        "googlemock/include",
+    ],
+    visibility = ["//visibility:public"],
+)
+
+cc_library(
+    name = "googletest_main",
+    srcs = ["googlemock/src/gmock_main.cc"],
+    visibility = ["//visibility:public"],
+    deps = [":googletest"],
+)
+
+cc_library(
+    name = "googletest_prod",
+    hdrs = [
+        "googletest/include/gtest/gtest_prod.h",
+    ],
+    includes = [
+        "googletest/include",
+    ],
+    visibility = ["//visibility:public"],
+)
+"""
+    native.new_git_repository(
+        name = "googletest_git",
+        build_file_content = BUILD,
+        commit = "d225acc90bc3a8c420a9bcd1f033033c1ccd7fe0",
+        remote = "https://github.com/google/googletest.git",
+    )
+
+    if bind:
+        native.bind(
+            name = "googletest",
+            actual = "@googletest_git//:googletest",
+        )
+
+        native.bind(
+            name = "googletest_main",
+            actual = "@googletest_git//:googletest_main",
+        )
+
+        native.bind(
+            name = "googletest_prod",
+            actual = "@googletest_git//:googletest_prod",
+        )
+
+def libevent_repositories(bind=True):
+    BUILD = """
+genrule(
+    name = "config",
+    srcs = glob([
+        "**/*",
+    ]),
+    outs = [
+        "config.h",
+    ],
+    tools = [
+        "configure",
+    ],
+    cmd = ' '.join([
```

Comment:
Buildifier: `s/' '/" "/g`.

Also, same comment regarding long strings in `cmd` as in nghttp2.